### PR TITLE
Add Integrations page

### PR DESCRIPTION
**Pull request by davidedistefano**
This PR adds a completely new page dedicated to Solidus Integrations.

The integrations are grouped into four categories: Payments, Logistics, Marketing, and Taxation. 

We'll probably add more services in the future.

## Preview
You can test this PR in the Netlify link below. Some content may change before the final launch. 

## ToDo
- [x] Add images
- [x] Fix links
- [ ] Add site title and meta description

### CODE REVIEW

File: source/integrations.html.erb [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+<div class="integration content-block" id="payments">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-md">
+        <%= inline_svg("icons/payments.svg", class: "isvg group") %>
+        <h2>Payments</h2>
+        <p>Choosing the right payment gateway can make a huge difference in how many customers you reach. Subscriptions, local payment methods, split payments… The possibilities are endless.</p>
+        <p>With its clean and flexible payments architecture, Solidus allows you to make it happen with no hassle. The foundation is already there for you, all you have to do is choose what’s best for your company.</p>
+        <a class="btn btn-primary" href="/extensions#extensions-group-payments" target="_blank">View extensions</a>
```
Comment (kennyadsl) [inline source/integrations.html.erb:44]: I'd go with a more specific CTA. What about `View all {group_name} extensions`?

File: source/integrations.html.erb [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        <a class="btn btn-primary" href="/extensions#extensions-group-marketing" target="_blank">View extensions</a>
+      </div>
+      <div class="integration__list col-md offset-md-1">
+        <div class="integration__list__item">
+          <div class="integration__list__item__image">
+            <%= inline_svg("integrations/google_analytics.svg", class: "isvg") %>
+          </div>
+          <div class="integration__list__item__description">
+            <h4>Google Analytics</h4>
+            <p>Easily integrate Solidus with Google Analytics.</p>
```
Comment (kennyadsl) [inline source/integrations.html.erb:139]: I think that having this not linking anyway is a blocking. We should at least have some resources about how to integrate GA in a Solidus store if we want to keep this link here.

File: source/integrations.html.erb [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+<div class="content-block mt-8">
+  <div class="container">
+    <div class="row">
+      <div class="col-md ">
+        <h2>We’re not done yet.</h2>
+        <p>We’ve just scratched the surface. Over the years, brands have integrated hundreds of services in their stores for the most disparate needs: advanced return processing, custom fraud detection, user-generated content, referral programs and a lot more.</p>
+        <p>Solidus allows your development team to quickly write integrations tailored to your unique needs that are stable and do the job, creating endless opportunities for you.</p>
+        <div class="integrations-cta">
+          <a class="btn btn-primary" href="/extensions">View extensions</a>
```
Comment (kennyadsl) [inline source/integrations.html.erb:216]: What about `View all extensions`?

File: source/integrations.html.erb (hunk 1 above)
Comment (davidedistefano) [reply source/integrations.html.erb:44]: I agree. Done!

File: source/integrations.html.erb (hunk 2 above)
Comment (davidedistefano) [reply source/integrations.html.erb:139]: The GA card is not a link though, but I agree it should be linked to a resource or an extension as the other services. Do we have a tutorial to share? Or do we have to create it?

Comment (kennyadsl) [reply source/integrations.html.erb:139]: It's new content, unfortunately we don't have it yet.

Comment (mfrecchiami) [reply source/integrations.html.erb:139]: Hi 👋 
I meddle in this conversation: we could add a tag like "Upcoming Integration", or something similar. Sounds good?

**Pull request merged**